server/src/entrypoint.sh

```diff
@@ -7,3 +7,9 @@ ln -s /srv/cvmfs /var/www/cvmfs
 a2enmod headers expires proxy proxy_http
```

Comment:
I can't find any references to the entrypoint.sh in this repo or in infra-config. Am I missing something or did I accidentally forget to delete this file?

Reply:
oops, you're right. I was trusting copilot too much lol.

entrypoint.sh doesnt seem to be used.